.github/workflows/deploy.yaml

```diff
@@ -80,3 +80,32 @@ jobs:
           --timeout 15m \
           ${{matrix.stage}}-energy-comparison-table \
           energy-comparison-table/infrastructure/app/chart/energy-comparison-table
+  slack-success:
+    name: Slack Notification (Success)
+    runs-on: ubuntu-22.04
+    needs: deploy-energy-supplier-comparison
+    steps:
+      - name: Notify Slack
+        uses: citizensadvice/message-slack-action@v1
+        with:
+          slack_bot_token: ${{ secrets.SLACK_BOT_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          channel_id: "C02K26YCER1" #content-platform_builds
+          title: ":rocket: Energy CSR Table Deployment Success :rocket:"
+          message: |
+            The Energy CSR Table has been successfully deployed to both QA and prod! | Please check your changes.
+  slack-failure:
+    name: Slack Notification (Failure)
+    runs-on: ubuntu-22.04
+    needs: deploy-energy-supplier-comparison
+    if: failure()
+    steps:
+      - name: Notify Slack
+        uses: citizensadvice/message-slack-action@v1
+        with:
+          slack_bot_token: ${{ secrets.SLACK_BOT_TOKEN }}
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          channel_id: "C02K26YCER1" #content-platform_builds
+          title: ":fire: Energy CSR Table Deployment Failure :fire:"
+          message: |
+            The Energy CSR Table has failed to deploy!
```

Comment:
Is there some way to include the environment in the message? I expect you'll want to look into production deployments much more urgently than dev deployments, in case it's broken in a way that affects the site.

Reply:
Or maybe the workflow name is part of the default message? (I just had a look at the [message-slack-action Readme](https://github.com/citizensadvice/message-slack-action))

Reply:
I'll have a play with this, yes. 

Because deployment is done via a matrix I think that the slack notification jobs will run after the last of the deployments (usually `prod`), so I think that would mean that we wouldn't get a notification for `dev` deployment unless it failed. Either way, I should be able to use `${{matrix.stage}}` to indicate whether the deployment was `dev` or `prod`.

Reply:
Now amended.